packages/host/tests/acceptance/code-patches-test.gts

```diff
@@ -424,6 +424,10 @@ ${REPLACE_MARKER}\n\`\`\``;
         submode: 'code',
         debug: false,
         openCardIds: [],
+        realmPermissions: {
+          canRead: true,
+          canWrite: true,
+        },
```

Comment:
this is was an upstream test failure that I started seeing after merging main into this branch.